server/src/test/java/io/crate/analyze/AlterTableRerouteAnalyzerTest.java

```diff
@@ -181,7 +205,7 @@ public void testRerouteCancelShardWithOptions() {
     }
 
     @Test
-    public void test_promote_replica_shard_with_literals() {
+    public void testPromoteReplicaShardWithLiterals() {
```

Comment:
Some time ago, we agreed on that using underscore_case results in better readability than the CamelCase style. Such we mostly use the underscore_case style when adding new test cases. This is not a strong requirement, but please don't change existing test names back to CamelCase.

Reply:
Should I name the new methods I added with the `_` scheme as well?